npm/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@ologbonowiwi/tailcall",
```

Comment:
```suggestion
  "name": "@tailcall/core",
```

Reply:
I've used `/tailcall` instead of `/core` because this will make the bin command be `tailcall`.